src/cloudant/view.py

```diff
@@ -451,10 +399,10 @@ def __call__(self, **kwargs):
             'use the database \'get_query_result\' convenience method.'
         )
 
-    def make_result(self, **options):
+    def custom_result(self, **options):
```

Comment:
I would, however, change the CloudantException message that is being raised by this method to something that tells the developer that the context manager is invalid rather than you cannot make a result.

Something like 'Cannot create a custom result context manager using a QueryIndexView.  If you wish to execute a query use the database "get_query_result" convenience method instead.'